packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -978,6 +978,9 @@ const schema: SchemaType<DbPost> = {
     }),
     optional: true,
     viewableBy: ['guests'],
+    insertableBy: ['admins', 'podcasters'],
+    editableBy: ['admins', 'podcasters'],
+    group: formGroups.audio,
```

Comment:
The code changes look good to me, but the form field might need to be forum gated - I'm not sure if we're adding audio on the EA Forum. Maybe @jpaddison3 can advise?

Reply:
We sometimes Forum gate the ability to add fields the other Forum would not care about, and sometimes we don't. I mildly would prefer it, but not strongly.

Reply:
The issue is that this adds an "Audio" option when creating a new post:
![Screenshot 2022-09-09 at 17 09 44](https://user-images.githubusercontent.com/5075628/189394377-dc625510-1fbd-45ad-9e0c-924c29820272.png)

Reply:
Is it auto-uncollapsed? Cause that would be annoying for anybody. Yeah, that upgrades me to medium caring

Reply:
It should only appear for admins and people you've manually added to the podcast group. (Although I agree it should probably be collapsed by default)

Reply:
I think even for admins, I don't like the extra category sitting unused on the EA Forum, and medium-care about it being Forum-gated. It's easy to do, I think.
```suggestion
    group: formGroups.audio,
    hidden: isEAForum,
```